Mathlib/Topology/Algebra/Nonarchimedean/Bases.lean

```diff
@@ -327,7 +327,7 @@ theorem nonarchimedean (hB : SubmodulesBasis B) : @NonarchimedeanAddGroup M _ hB
 
 library_note "nonarchimedean non instances"/--
 The non archimedean subgroup basis lemmas cannot be instances because some instances
-(such as `MeasureTheory.AEEqFun.instAddMonoid` or `topological_add_group.to_has_continuous_add`)
+(such as `MeasureTheory.AEEqFun.instAddMonoid` or `TopologicalAddGroup.to_hasContinuousAdd`)
```

Comment:
This comment is still wrong, I think: the API docs don't contains the new name. Help welcome.

Reply:
It's just a structure projection, so it should be: 
`TopologicalAddGroup.toContinuousAdd`

Reply:
:bulb: Thanks!
```suggestion
(such as `MeasureTheory.AEEqFun.instAddMonoid` or `TopologicalAddGroup.toContinuousAdd`)